mixer/pkg/runtime/config/ephemeral.go

```diff
@@ -285,7 +287,6 @@ func (e *Ephemeral) processStaticAdapterHandlerConfigs(ctx context.Context) map[
 				staticConfig.Params = c
 			}
 			handlers[key.String()] = staticConfig
-			stats.Record(ctx, monitoring.HandlersTotal.M(1))
```

Comment:
I think this needs an error counter incremented for L267

Reply:
I'm confused about why this should be an error counter.

We've found the static config for the handler and have added it to `handlers` (which is returned). Later, we increment the counter based on len(handlers). What have I missed?

Reply:
Yes, my comment doesn't make sense in this context. Maybe I was looking at a different diff or something.